runtime/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "saxaboom-runtime"
+version = "0.0.0+irconverter-1.1"
+authors = ["Traverse Research <support@traverseresearch.nl>"]
+edition = "2021"
+license = "Apache-2.0"
+homepage = "https://traverseresearch.nl"
+repository = "https://github.com/Traverse-Research/saxaboom"
+description = "Binding library for metal_irconverter"
+categories = ["api-bindings", "external-ffi-bindings", "graphics"]
+keywords = ["metal", "shader", "metal_irconverter", "apple"]
+
+[package.metadata.docs.rs]
+targets = [
+  "aarch64-apple-darwin", # presented first in docs.rs, keep this here
+  "aarch64-apple-ios",
+  "aarch64-apple-ios-sim",
+  "x86_64-apple-darwin",
+  "x86_64-apple-ios",
+]
+
+[dependencies]
+metal = "0.28"
```

Comment:
We just merged [the `gpu-allocator` update for `.0.29`](https://github.com/Traverse-Research/gpu-allocator/commit/db51810f7b54dd36e073ae1c6315281080a650b3), let's get this bumped here too and we'll bump `breda` at once :ok_hand: